source/Files.cpp

```diff
@@ -554,24 +381,23 @@ void Files::Write(FILE *file, const string &data)
 
 
 
-void Files::CreateFolder(const std::string &path)
+void Files::CreateFolder(const filesystem::path &path)
 {
 	if(Exists(path))
```

Comment:
```suggestion
	if(exists(path))
```

Reply:
Everything else uses the method, not the stdlib function